src/main/java/com/quantumsoft/qupathcloud/converter/dicomizer/Wsi2dcmLibrary.java

```diff
@@ -14,34 +14,43 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 package com.quantumsoft.qupathcloud.converter.dicomizer;
+
 import com.sun.jna.Library;
 import com.sun.jna.Native;
-import com.sun.jna.NativeLibrary;
 import com.sun.jna.NativeLong;
-import com.sun.jna.Pointer;
-import com.sun.jna.ptr.DoubleByReference;
 import java.nio.ByteBuffer;
 import java.nio.DoubleBuffer;
+
 /**
- * JNA Wrapper for library <b>wsi2dcm</b><br>
- * This file was autogenerated by <a href="http://jnaerator.googlecode.com/">JNAerator</a>,<br>
- * a tool written by <a href="http://ochafik.com/">Olivier Chafik</a> that <a href="http://code.google.com/p/jnaerator/wiki/CreditsAndLicense">uses a few opensource projects.</a>.<br>
- * For help, please visit <a href="http://nativelibs4java.googlecode.com/">NativeLibs4Java</a> , <a href="http://rococoa.dev.java.net/">Rococoa</a>, or <a href="http://jna.dev.java.net/">JNA</a>.
+ * The interface wsi2dcm library.
```

Comment:
It would be nice to add some details on what exactly this interface does..

Reply:
Done